flake-modules/dev/init-script/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "nixvim-init"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+clap = "4.5.20"
```

Comment:
I'd suggest using `clap = { version = "4.5.20", features = ["derive"] }`, it's quite a bit more ergonomic

Reply:
Ok thanks ! Actually, I don't even think that we will need clap at all.
Good to know though.

Reply:
> Actually, I don't even think that we will need clap at all.

I think we'll probably end wanting some decent argument parsing and help-text at some point...

Even if we have a mostly interactive UX it'd still be nice to be able to do things via CLI args.